### PR TITLE
Batch update

### PR DESCRIPTION
**Pull request by thoemmi**
As requested in #37, this PR introduces the static method `Resource.Save(string filename, IEnumerable<Resource> resources)` for saving multiple resources in one batch.

For whatever reason the implementation of `IconImageResource` defers from other resources, so I had to add a special code path for saving those resources.

### CODE REVIEW

**Comment by dblock** [comment]
Good. This needs tests and a CHANGELOG entry please.

**Comment by thoemmi** [comment]
I've added a test and an entry to CHANGELOG.

**Comment by thoemmi** [comment]
The last line in the AppVeyor log says:

> Uploading artifact target\Release\Vestris.ResourceLib.1.5.78.0.nupkg (68.2 KB)...Error uploading artifact to Google Storage.

so the failing build is not a result of my changes. I guess it's caused by the AppVeyor's project configuration.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### v.Next
```
Comment (dblock) [inline CHANGELOG.md]: Maybe say 1.6 (Next), no big deal.

File: Source/ResourceLibUnitTests/ResourceTests.cs [hunk 2]
```diff
@@ -5,6 +5,7 @@
 #endregion
 
 using System.Collections.Generic;
+using System.Linq;
```
Comment (dblock) [inline Source/ResourceLibUnitTests/ResourceTests.cs]: Is this used anywhere below?

**Comment by dblock** [comment]
@thoemmi @icnocop Can you help me out with the nuget upload issue? I am not sure what I need to do wrt keys here?

File: Source/ResourceLibUnitTests/ResourceTests.cs (hunk 2 above)
Comment (thoemmi) [inline Source/ResourceLibUnitTests/ResourceTests.cs]: Sorry, this is an artifact from an earlier implementation of mine. I'll delete it.

**Comment by dblock** [comment]
Rebase and squash your commits, too, please.

**Comment by thoemmi** [comment]
@dblock The last time it was me who uploaded the NuGet package. You can see [on NuGet](https://www.nuget.org/packages/Vestris.ResourceLib) that I'm the owner of the package.

**Comment by dblock** [comment]
Is it as simple as adding me to the owners? I created a `dblock` account on NuGet.

**Comment by dblock** [comment]
And now that succeeded? I don't get it, does it just work 50% of the time? :)

**Comment by dblock** [comment]
I'll merge this, it's all good.

**Comment by thoemmi** [comment]
Uups, I just squashed the commits.

**Comment by thoemmi** [comment]
@dblock, I added you as an owner at NuGet. But wait before releasing 1.6, I've found another bug. However, I don't have time today to write proper unit tests.

**Comment by dblock** [comment]
You're the one releasing it, so you drive :)

**Comment by icnocop** [comment]
Let me know if you guys still need my help. Thank you! :)

**Pull request merged**